README.md

```diff
@@ -88,8 +88,11 @@ Once inside, you have many commands at your disposal. Here are a few with some d
 Fully rebuild from the remote openAPI schema
 `make build-from-schema`
 
-Fully rebuild from current source code
-`make rebuild-from-source`
+Build with current source code
+`make build-from-source`
+
+Fix linting/style check in files
+`make fix-linting`
```

Comment:
Does it need to be explicitly mentioned now? Why wasn't it before?

Reply:
Previously, the `rebuild-from-source` did both steps. But since now we don't really build a new client but just move it, I've added both steps to the README separately.

Reply:
Does this task actually need to be called by us manually at any stage?

Reply:
If you manually change the client you should call this to make sure linting is correct. But I see your point, this is marginally useful as a manual command atm. I'll remove it and leave only the change to `build_from_source`